docs/spec-files/message_structure.md

```diff
@@ -86,8 +88,6 @@ Headers in [DIDComm Messaging](.) are intended to be extensible in much the same
 - **type** - REQUIRED. Plaintext message type, useful for message handling in application-level protocols. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri), that when resolved gives human readable information about the message. The attribute's value SHOULD predict the content in the `body` of the message.
 
 - **typ** - OPTIONAL. Media type of the JWM content.
-
-- **to** - OPTIONAL. Identifier(s) for recipients. MUST be an array of strings where each element is a valid [DID](https://www.w3.org/TR/did-core/) or [DID URL](https://w3c.github.io/did-core/#did-url-syntax) (without the [fragment component](https://w3c.github.io/did-core/#fragment)) that identifies a member of the message's intended audience.
```

Comment:
I'd prefer to see us move the test into this section here to remain consistent in our definition of headers rather than move the normative requirements into a separate section which doesn't call out that it's making requirements around the usage of the `to` header.

Reply:
This is just removing the duplicated entry for the "to" header. This was duplicated in #277 without reason.

I think it was a copy pasta mistake.

If you look below in that file, it's still containing this exact same paragraph.

Reply:
It seems @TelegramSam is also fixing this in his latest PR #274.
So I guess I can drop this removal from this PR.

Reply:
Thanks for pointing that out, I didn't realize this.